src/Joomlatools/Console/Command/Database/Install.php

```diff
@@ -249,14 +249,26 @@ protected function _getInstallFiles($sample_data = false)
                 $path = $this->target_dir.'/installation/sql/mysql/';
             }
 
-            $files[] = $path.'joomla.sql';
+            $version = Util::getJoomlaVersion($this->target_dir);
+
+            if (version_compare($version->release, '3.9', '>'))
```

Comment:
@yiendos This should be greater or equal than 4.0, because now it's also using these SQL files for 3.9.x versions, which breaks installation.

Reply:
@stevenrombauts great catch that was complete fubar... I've just run over the alpha => beta installs again today and turns out that only the beta versions have the new sql files... and no doubt in the future the dev team will roll back to the single joomla.sql file. So I've put a catch in for this as well. Let me know what you think